static/docs/commands-reference/remote_modify.md

```diff
@@ -228,6 +228,30 @@ For more information about the variables DVC supports, please visit
 
 </details>
 
+<details>
+
+### Click for Aliyun OSS available options
+
+- `oss_key_id` - oss key id to use to access a remote.
+
+  ```dvc
+    $ dvc remote modify myremote oss_key_id my-key-id --local
```

Comment:
remove indents in code blocks

Reply:
@shcheklein Done. Indents were added to match previous sections in the file. Though I have removed indents in mine.